upgrade/capsule.py

```diff
@@ -57,6 +60,10 @@ def satellite6_capsule_setup(sat_host, os_version, upgradable_capsule=True):
         execute(delete_rhevm_instance, cap_instance)
         logger.info('Turning on Capsule Instance ....')
         execute(create_rhevm_instance, cap_instance, cap_image)
+        for cap_host in cap_hosts:
+            if not host_pings(cap_host):
+                sys.exit(1)
```

Comment:
1. Check for all the capsules in list
2. Print what and all capsules are not pinging and 
3. then exit.